### DISCUSSION TITLE
How should one set COLDSTARTDATE for TC hindcasts?

### DISCUSSION THREAD

**Discussion by notstarboard** [StormSurgeLive/asgs#1125] [Q&A] [answered]
My assumption has been that one would want to set COLDSTARTDATE such that COLDSTARTDATE + HINDCASTLENGTH = [time of the first advisory you want to run with]. When discussing with Isaac and Deb, though, they had a recollection of there being a 6-hour offset because ASGS automatically builds in some sort of short spin-up with observed winds between the main hindcast for tidal spinup to hopefully get more accurate initial conditions. I've never had a good understanding of what the "nowcast" is, but I'm assuming this is what they're referring to. 

I could see using something like COLDSTARTDATE + HINDCASTLENGTH + [Nowcast length (6 hours?)] = [time of the first advisory you want to run with], but when I check nowcast/run.properties RunStartTime and RunEndTime are exactly the same. This seems to imply there's no need to add time for the nowcast, so I'm back to thinking I should the original formula I had assumed. Am I thinking about this right?

**Comment by jasonfleming** [comment, marked as answer]
Hey @notstarboard you may want to check out our [wiki for hindcasting historical storms with ASGS](https://github.com/StormSurgeLive/asgs/wiki/Hindcasts).

**Reply by notstarboard** [reply]
Ooh that's perfect - thanks! Didn't realize / forgot that page existed. I was going off the main operators guide, which has some detail, but which I don't think explains this specific piece.
